decidim-core/app/cells/decidim/activity_cell.rb

```diff
@@ -90,6 +90,15 @@ def element_id
       "action-#{model.id}"
     end
 
+    def cache_hash
+      hash = []
+      hash << model.class.name.underscore
+      hash << I18n.locale.to_s
```

Comment:
Since #7943 we don't need to specify the locale in the cache key! 😄 Could you remove it, please? @alecslupu

Reply:
@mrcasals Changes done. Branch updated with the latest changes on develop.